pkg/apis/servicecatalog/types.go

```diff
@@ -46,15 +46,11 @@ type ClusterServiceBrokerList struct {
 	Items []ClusterServiceBroker
 }
 
-// ClusterServiceBrokerSpec represents a description of a Broker.
-type ClusterServiceBrokerSpec struct {
-	// URL is the address used to communicate with the ClusterServiceBroker.
+// SharedServiceBrokerSpec represents a description of a Broker.
+type SharedServiceBrokerSpec struct {
```

Comment:
I think we have agreed in Slack to use `Common` or `Core` for shared embedded struct naming?